README.md

```diff
@@ -11,3 +11,14 @@ In the project directory, you can run:
 npm install
 npm run start
 ```
+
+## Add this app to your pod using SAI
+In order to add this app to your pod:
+* use [this branch of SAI](https://github.com/michielbdejong/sai-js/tree/investigating-77), and run it on localhost according to the instructions there
+* run this app according to the instructions above
+* go to http://localhost:4200/applications
+* click '+ ADD APPLICATION'
+* fill in http://localhost:3600 as the Client ID
```

Comment:
```suggestion
* login at http://localhost:4200/ and connect server
* login at  http://localhost:3600 and initialize authorization
``` 

you can find out how to do it at https://github.com/janeirodigital/sai-js/blob/main/examples/vuejectron/src/store/core.ts#L53-L59

Reply:
Those instructions will fail unless you also add that code into this app, right?
Does the `+ ADD APPLICATION` button not work?

Reply:
It does; you still need a valid Client ID document https://github.com/janeirodigital/sai-js/issues/77#issuecomment-1978593085

I still think requesting authorization from the app is more intuitive than making sure one passes a valid client ID into the input field. But since currently we talk about app developers it should not matter.